baseplate/metrics.py

```diff
@@ -313,7 +313,7 @@ def replace(self, new_value):
 def make_client(namespace, endpoint):
     """Return a configured client.
 
-    :param str namespace: The root key to namespace all metrics under.
+    :param str namespace: The root key to prefix all metrics with.
```

Comment:
You've translated "namespace" to "prefix" here from the function spec to the comment about what it is.
Should the parameter also be called "prefix"? I think we could either "namespace" a set of things or "prefix" some labels, but with this change you're doing both. "Namespace" implies something deeper than just attaching a prefix to labels.